.gitignore

```diff
@@ -169,4 +169,4 @@ data/
 test.py
 **.pt
 **.pth
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
Hi @onuralpszr why do we need the `.DS_Store` file in this case?

Reply:
I am ignoring it ? I didn't understand your question